WalletWasabi.Tests/RegTests.cs

```diff
@@ -3403,7 +3403,7 @@ public async Task CoinJoinMultipleRoundTestsAsync()
 					}
 				}
 				SmartCoin[] unspentChanges = wallet.Coins.Where(x => x.Label == "ZeroLink Change" && x.Unspent).ToArray();
-				await wallet.ChaumianClient.DequeueCoinsFromMixAsync(unspentChanges);
+				await wallet.ChaumianClient.DequeueCoinsFromMixAsync(unspentChanges, string.Empty);
```

Comment:
I prefer `""` to `string.Empty`, because it's more widely used and compact.

```suggestion
				await wallet.ChaumianClient.DequeueCoinsFromMixAsync(unspentChanges, "");
```

Reply:
ack